### ISSUE TITLE
Failures on Initial test_all.dart, the dart bots test all the files ending in "_test.dart", this file allows developers to easily run all the tests themselves.

### ISSUE THREAD

**Issue by whesse** [dart-lang/sdk#40652]
This failure occured a few commits after the failures on two other new dartdev/test/commands/ tests, which also failed on the dart2js-minified-d8 configuration, and were approved by @jwren  

There are new test failures on [Initial test_all.dart, the dart bots test all the files ending in "_test.dart", this file allows developers to easily run all the tests themselves.](https://github.com/dart-lang/sdk/commit/c856de0747b3ddffcc1c2e116b4bd6e761126201).

The tests
```
pkg/dartdev/test/commands/flag_test RuntimeError (expected Pass)
```
are failing on configurations
```
dart2js-minified-linux-d8
```

**Comment by jwren** [comment]
The `dart2js-...-minified-...` bots seem to be passing on the bots at this point, please re-open with steps to reproduce if something is broken (but still green?) on the bots.

**Comment by whesse** [comment]
The bot greenness only reflects whether all failures have been marked as approved on the results feed.  You have to look at the logs output to see if the test is still failing.
The log link from the results feed failure ( on Jan 27) shows the failure:
https://dart-ci.appspot.com/log/dart2js-minified-strong-linux-x64-d8/dart2js-minified-linux-d8/8261/pkg/dartdev/test/commands/flag_test

The log link for today (replace the build number with 'latest') shows the test not failing.  But you can't use builder greenness to tell you that.
https://dart-ci.appspot.com/log/dart2js-minified-strong-linux-x64-d8/dart2js-minified-linux-d8/latest/pkg/dartdev/test/commands/flag_test

**Comment by jwren** [comment]
Very confusing.  Clicking the second link yeilds `No log for test...`

Since the greenness on the buildbot isn't an indicator, is there some set of URLs that Dart SDK developers click through to look to see if things are breaking?

**Comment by whesse** [comment]
Don't really know what you mean by "are breaking".  but it is clear that we
need an easy way to check the status of a test, or of a test on a
configuration. Will definitely make that a priority in the next month or
so.that one unapproved test failure was making the builder read for about 2
weeks, so I finally approved it, and filed an issue for someone to look
into it more,. if the two other tests I mentioned that were approved by you
were fixed in some way by skipping them then it would have been to check
that the same thing was done with this test. Again this would have been
this would have been much easier if it have been done closer to the time of
the original test failure. Sorry I'm doing this by voice in a loud bar.
